patches/fontconfig/0001-Add-Managarm-support.patch

```diff
@@ -0,0 +1,26 @@
+From 817dc2102610dd853a718e14eee63373ab31c8eb Mon Sep 17 00:00:00 2001
+From: Dennis Bonke <admin@dennisbonke.com>
+Date: Mon, 29 Aug 2022 01:04:27 +0200
+Subject: [PATCH] Add Managarm support
+
+Signed-off-by: Dennis Bonke <admin@dennisbonke.com>
+---
```

Comment:
is there a matching revbump?

Reply:
Nope, and it isn't needed. Fontconfig got updated in #206 and never build successfully on the ci.

Reply:
ah, fair enough